packages/testcontainers/src/containers/DeFiDContainer.ts

```diff
@@ -36,7 +36,7 @@ export abstract class DeFiDContainer extends DockerContainer {
     if (process?.env?.DEFICHAIN_DOCKER_IMAGE !== undefined) {
       return process.env.DEFICHAIN_DOCKER_IMAGE
     }
-    return 'defi/defichain:3.2.0'
+    return 'defi/defichain:HEAD-fe4ccb39d'
```

Comment:
Btw, is this update included in `3.2.1`? If so, we can wait for https://github.com/JellyfishSDK/jellyfish/pull/1955 to be merged first then we can rebase/merge

Reply:
Hmm, the https://github.com/DeFiCh/ain/pull/1666 PR is not merged yet. Its going to be included in `3.2.2` there i guess as it tagged.

Reply:
Let's wait for `3.2.2` since we need `3.2.1` in first. I assume the features released in `3.2.1` aren't included in the `HEAD-*` you're using.

Reply:
Updated to the latest image: [`0a7aeb4`](https://github.com/JellyfishSDK/jellyfish/pull/1943/commits/0a7aeb417016e8d9dea9b48fe8fce75ccaf038cb)